tests/pytorch_tests/function_tests/test_fully_quantized_exporter.py

```diff
@@ -100,41 +100,28 @@ def test_layers_wrapper(self):
         self.assertTrue(self.fully_quantized_mbv2.features_0_0_bn.is_weights_quantization)
         self.assertFalse(self.fully_quantized_mbv2.features_0_0_bn.is_activation_quantization)
 
-        self.assertTrue(isinstance(self.fully_quantized_mbv2.features_0_2, PytorchQuantizationWrapper))
-        self.assertTrue(isinstance(self.fully_quantized_mbv2.features_0_2.layer, torch.nn.ReLU6))
-        self.assertFalse(self.fully_quantized_mbv2.features_0_2.is_weights_quantization)
-        self.assertTrue(self.fully_quantized_mbv2.features_0_2.is_activation_quantization)
```

Comment:
Check that relu6 activations are quantized

Reply:
already did in this test "test_activation_qc"